mobile/library/common/jni/BUILD

```diff
@@ -95,7 +93,7 @@ android_debug_info(
 ## Targets for local execution
 # OS X binary (.jnilib) for NDK testing
 envoy_mobile_so_to_jni_lib(
-    name = "libndk_envoy_jni.jnilib",
```

Comment:
pretty sure this name is important for testing on macOS, @jpsim or @Augustyniak can verify

Reply:
Seeing as how CI runs tests that go through the JNI on macOS, I think that's sufficient coverage, but I'll test locally to be sure.

Reply:
`./bazelw test --define=signal_trace=disabled //test/java/...` still passes for me locally on macOS.